components/Sidebar/HeaderTab.tsx

```diff
@@ -47,31 +54,59 @@ const HeaderTab = ({
           sx={{
             width: DRAWER_WIDTH,
             boxShadow: 'none',
-            borderRadius: 0,
           }}
         >
           <AccordionSummary
-            expandIcon={hasSubheaders && <ArrowDropDownIcon />}
+            expandIcon={isSidebarOpen && hasSubheaders && <ArrowDropDownIcon />}
             aria-controls="panel1bh-content"
             id="panel1bh-header"
             sx={{
-              '&.MuiButtonBase-root.MuiAccordionSummary-root': {
-                cursor: hasSubheaders ? 'pointer' : 'default',
-              },
-              '&.MuiButtonBase-root.MuiAccordionSummary-root:hover': {
-                cursor: hasSubheaders ? 'pointer' : 'default',
+              '& .MuiAccordionSummary-content': {
+                justifyContent: isSidebarOpen ? 'normal' : 'center',
               },
               backgroundColor:
-                tabSelected === headerData.title ? MR_GRAY_1 : 'white',
+                tabSelected === headerData.title
+                  ? SELECTED_TAB_COLOR
+                  : UNSELECTED_TAB_COLOR,
+              width: DRAWER_WIDTH,
             }}
           >
-            <Typography
-              sx={{
-                color: 'text.primary',
+            <div
+              style={{
+                width: '20px',
+                height: '20px',
```

Comment:
change `width` to `minWidth` because in the expanded view the user account icon is smaller than 20px